server/Route.js

```diff
@@ -1,5 +1,55 @@
 const express = require('express');
+const db = require('./db/index');
 
 const router = express.Router();
```

Comment:
@LittleJennie 

I love that you are using express Router--If I might make one recommendation--> as this file grows, you may consider having separate files for the routes, and importing them to a single index.js file (if your implementation grows large enough).